test/parallel/test-process-env.js

```diff
@@ -67,3 +67,12 @@ assert.equal(3, date.getUTCHours());
 assert.equal(5, date.getHours());
 */
 /* eslint-enable max-len */
+
+// environment variables should be case-insensitive on Windows, and
+// case-sensitive on other platforms
+process.env.TEST = 'test';
+if (common.isWindows) {
```

Comment:
I think you’ll need to change `require('../common');` to `const common = require('../common');` for this to work